hbase-shell/src/test/ruby/hbase/replication_admin_test.rb

```diff
@@ -575,7 +575,7 @@ def assert_tablecfs_equal(table_cfs, table_cfs_map)
     end
 
     define_test "set_peer_bandwidth: works with peer bandwidth upper limit" do
-      cluster_key = "localhost:2181:/hbase-test"
+      cluster_key = MiniZookeeperCluster::HOST + ":2181:/hbase-test"
```

Comment:
Test failure looks related to this one. Does the `MiniZookeeperCluster` need initialization before?

Reply:
Thank you for taking a look @HorizonNet  ... Let me see.

Reply:
s/MiniZookeeperCluster::HOST/MiniZooKeeperCluster::HOST/

... the 'k' capitialization. Joy!